src/test/java/io/vertx/test/core/NetTest.java

```diff
@@ -1261,6 +1288,20 @@ public void testSpecificTlsProtocolVersion() throws Exception {
   }
 
   @Test
+  // test client SSL handshakeTimeout option
+  public void testClientSslHandshakeTimeout() throws Exception {
+    testTLS(Cert.NONE, Trust.NONE, Cert.SERVER_JKS, Trust.NONE, false, true, true, false, new String[0],
```

Comment:
I think SSL timeout should have specific test and not reuse the general one, for instance for server:

- setup an SSL server with a timeout with a small value for instance 400ms
- run a NetClient that connects to the server and does nothing
- wait for the SSL timeout to fail, i.e the client will get an error on connect

Reply:
Thanks for your helpful suggestions! I'm working on it